keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -661,7 +664,71 @@ Recall that end users cannot usually observe component logs or access metrics.
   - Other field:
 - [ ] Other (treat as last resort)
   - Details: -->
-N/A
+
+- [x] Other (treat as last resort)
```

Comment:
The question is not about a testcase. It's about how they know if the feature is working when they have configured inclusion policy.

The reality is that they can only observe the behavior based on the existing pods.